client/components/site-verticals-suggestion-search/utils.js

```diff
@@ -0,0 +1,59 @@
+/** @format */
```

Comment:
I think this whole module be deprecated now?

Reply:
Ah good call!